### PR TITLE
Maintenance doc

### PR DESCRIPTION
**Pull request by morganridel**
Some explanation on how to do maintenance with the current method

### CODE REVIEW

**Comment by pyphilia** [comment]
LGTM but since I know well the process, I'll let others read it as well. @dialexo @spaenleh

File: docs/maintenance.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      return event.Records[0].cf.request;
+    }
+
+    return {
+      status: '302',
+      statusDescription: 'Found',
+      headers: {
+        'location': [{
+          key: 'Location',
+          value: 'https://maintenance.dev.graasp.org',
```
Comment (spaenleh) [inline docs/maintenance.md:52]: Maybe forgot to change this to be aligned with the below paragraph.

File: docs/maintenance.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Put Graasp in maintenance mode
+
+Maintenance mode is currently setup at the infrastructure level. Go through the following steps to enable the maintenance:
+
+1. In Cloudfront, go into each distribution frontend you want to be affected
+2. Behavior > Edit the default behavior > Function association > Viewer request > Lambda@Edge > `arn:aws:lambda:us-east-1:592217263685:function:maintenance-check:2`
```
Comment (spaenleh) [inline docs/maintenance.md:6]: Is there a reason why the lambda is in the `us-east-1` region ? Or am I missing something ?

**Review by spaenleh — APPROVED** [review]
🤩 Looks good to me ! Thank you for a detailed explanation with examples, and reasoning as well as alternatives 👍 

If you can, maybe just add a schema/diagram for visual explanation 🎨 (would be the cherry on top 😉) You can use [mermaidjs directly in Github Markdown](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams) or insert an image 🤓

**Pull request merged**